src/test/java/org/folio/edge/patron/MainVerticleTest.java

```diff
@@ -1197,5 +1256,16 @@ private void validateHolds(Hold expectedHolds, Hold actualHolds) {
     //check that the actual timestamp is within a minute of the expected timestamp
     assertTrue((actualRequestDateTs - expectedRequestDateTs)/1000 < 60);
   }
+
+  private String swapExpirationDate(String jsonRequestMessage, String dateString) {
+    final String regex = "(?<=\"expirationDate\").*\".*\"";
+    final Pattern pattern = Pattern.compile(regex);
+    final Matcher matcher = pattern.matcher(jsonRequestMessage);
+
+    if (matcher.find()) {
+      jsonRequestMessage = jsonRequestMessage.replaceAll("(?<=\"expirationDate\").*\".*\"", dateString);
```

Comment:
💥 <-- My head. Isn't the string in `replaceAll` the same as the `regex` string above? Do we need to do the find? couldn't we just do the replaceAll? I can't wait until this code goes!!! 😄 Nothing against your code, I just think this whole EDS work around is a hack. A necessary hack, but yuck... 🤢

Reply:
true, no need to check for it. I'll remove the check.